node/pkg/driver/device_connectivity/device_path_s390x.go

```diff
@@ -1,4 +1,4 @@
 package device_connectivity
 
 // on Z-Systems, the udev subsystem is a Channel Command Word
-const fcSubsystem = "ccw"
```

Comment:
also here comment is not needed

Reply:
the comment is not part of this PR (and you approved it in https://github.com/IBM/ibm-block-csi-driver/pull/124).
then why is it needed in Power?

Reply:
I would remove it in power too.

Reply:
done